shared/validation/languages/it.js

```diff
@@ -0,0 +1,47 @@
+// Minimum of words that qualify as a sentence.
+const MIN_WORDS = 1;
+
+// Maximum of words allowed per sentence to keep recordings in a manageable duration. 
+// Italian: finché non abbiamo il controllo sui caratteri, o quello che tiene conto della presenza di numeri, metto 17 parole che dovrebbero essere salvo rarissimi casi sempre entro il limite di 125 caratteri. 
+const MAX_WORDS = 17;
+
+// Numbers that are not allowed in a sentence depending on the language. For
+// Italian: tutti i numeri sono consentiti ma se e quando avremo la possibilità di fare regole personalizzate metteremo una condizione ulteriore sulla lunghezza se vi sono dei numeri nella frase.
+const NUMBERS_REGEX = /[0-9]+/;
+
+/* eslint-disable-next-line no-useless-escape */
+// Italian: Simboli non permessi, aggiungere anche qui sotto oltre che nella regex:
+// “ ” ‘ ’ ( ) É 
+//doppio " " e più di un "." nella stessa frase.
+const SYMBOL_REGEX = /[“”‘’()É]| {2,}|\..*\./;       //  original:   /[<>\+\*\\#@\^\[\]\(\)\/]/;
+// Any words consisting of uppercase letters or uppercase letters with a period
+// inbetween are considered abbreviations or acronyms.
+// This currently also matches fooBAR but we most probably don't want that either
+// as users wouldn't know how to pronounce the uppercase letters.
+//Versione italiana: dag7dev
+const ABBREVIATION_REGEX = /[A-Z][a-z]{2,}|[A-Z][a-z]+\.*[A-Z][a-z]+/;
```

Comment:
If I read this correctly (haven't tested on the website yet), this would also filter out `Test` due to the `[A-Z][a-z]{2,}` part which basically matches any word starting with a capital letter and having two following lowercase letters. Is there a reason you included `[a-z]` in this regex?

Reply:
This section was written by @dag7dev, so I leave the final decision to him, but I think his intention was to also filter abbreviations with lowercase letters. Which would be kinda hard to do in my opinion with the mechanism that was used for the english version, so it's probably better to stick to that, but let's see if our colleague has a better idea. 

In the case of abbreviations with a `.` they'll already be filtered by the symbols regex which disallows a `.` inside the sentence other than the final one.

Reply:
hey @MichaelKohler, as @jotaro-sama saied I wanted to filter abbreviations with lowercase. I have changed the regex as you suggested! Thanks!